packages/block-library/src/block/edit.js

```diff
@@ -278,6 +314,15 @@ export default function ReusableBlockEdit( {
 		[ editedRecord.blocks, editedRecord.content ]
 	);
 
+	// A map of clientIds to the old nano id system to provide back compat.
+	const legacyIdMap = useRef( {} );
+	useEffect( () => {
```

Comment:
Nit: I don't think `initialBlocks` would ever change. Maybe we can move this to the `useEffect` where we initially use it to replace the inner blocks (L327).

Reply:
Thanks, I've done that 👍